src/components/pages/AdminDashboard/ModerationTools.js

```diff
@@ -60,13 +60,15 @@ const ModerationTools = () => {
   };
 
   //ERRLOG: no data in response
```

Comment:
both functions below are not called - instead of removing them I asked the linter to ignore the fact that they are unused as they may need to be implemented at some point.